packages/flutter/lib/src/material_animated_icons/data/arrow_menu.g.dart

```diff
@@ -0,0 +1,1027 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// AUTO GENERATED FILE DO NOT EDIT!
+// This file was generated by svg2dart.
```

Comment:
regenerate the files now you've updated the tool

Reply:
done